### DISCUSSION TITLE
installation on 1gb Ram VPS

### DISCUSSION THREAD

**Discussion by hitzgh** [AzuraCast/AzuraCast#5789] [General]
Hello, i know the recommended RAM for azuracast is 2gb but i wanted to ask, is it possible to run azuracast on 1gb of ram vps

**Comment by ItsMitchh** [comment]
Will it work, probably
Is it advised? Not really.

**Reply by hitzgh** [reply]
I tried but I keep getting error

**Comment by krono-i2** [comment]
You can, I used to run my instance on a similar machine.
I have to used Ansible installation.

**Reply by hitzgh** [reply]
i tried the server was too slow to run
